### DISCUSSION TITLE
Translate from foreign text to English

### DISCUSSION THREAD

**Discussion by c-nemo-y** [openai/whisper#36] [Q&A] [answered]
Hey, the use case for the whisper model seems to only accept audio and video inputs, would it be possible to ingest foreign text files and directly translate them to English text?

**Comment by jongwook** [comment, marked as answer]
Not out of the box. A very roundabout way is to feed the text to a TTS engine and then translate it using Whisper, but it's probably better off using Google Translate or something.

**Reply by koreanmarine** [reply]
안녕하세요. 그럼 혹시 whisper안의 기능으로 한국어 speech를 English로 바꾸는 것보다  한국어 text로 출력한 다음 다른 모델을 사용해서 english로 바꾸는게 더 효율적이라는 말씀이신가요?
